src/transformers/models/donut/configuration_donut_swin.py

```diff
@@ -13,6 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """ Donut Swin Transformer model configuration"""
+from collections import OrderedDict
+from typing import Mapping
+
+from transformers.onnx import OnnxConfig
```

Comment:
Please use relative imports instead of absolute ones:

```suggestion
from .onnx import OnnxConfig
```